Fix Ubuntu 18.04 molecule failures

jsf9k opened the pull request:
## 🗣 Description ##

This pull request adds the `netbase` package for Ubuntu, which fixes a reproducible error in the `converge` state of `molecule` for Ubuntu 18.04.  The `geerlingguy/docker-ubuntu1804-ansible:latest` Docker image is suddenly missing this package, perhaps due to an omission in its base image.

This pull request also cleans up a few files, removing some now-unnecessary cruft.

## 💭 Motivation and Context ##

The `molecule` tests have been reproducibly failing for this repository lately, and this PR fixes that.

This PR resolves (partially) #30.

## 🧪 Testing ##

All `pre-commit` hooks and `molecule` tests pass.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

jsf9k committed:
Remove Kali-specific symlink to Debian.yml

Ansible now correctly identifies Kali as belonging to the Debian Linux
family.

jsf9k committed:
Remove comments that add no information

jsf9k committed:
Add the netbase package for Ubuntu

This package is missing in the geerlingguy/docker-ubuntu1804-ansible:latest Docker image.

jsf9k committed:
Remove now-unnecessary regex_replace() foo

Ansible now correctly identifies Kali Linux as being the Kali
distribution of Debian Linux, so this filtering is no longer
necessary.

dav3r reviewed (approved):
Fine detective work! 🥇 
Did you mention anything about this to geerlingguy?

jsf9k commented:
> Did you mention anything about this to geerlingguy?

I think the `netbase` package is missing from the upstream Docker image, and therefore outside his control, but I'll mention @geerlingguy here so he knows.

mcdonnnj commented on tasks/main.yml:
```diff
@@ -1,17 +1,12 @@
-        - "{{ ansible_distribution | regex_replace('[ /]', '_') }}_{{ ansible_distribution_version }}.yml"
-        - "{{ ansible_distribution | regex_replace('[ /]', '_') }}.yml"
-        - "{{ ansible_os_family | regex_replace('[ /]', '_') }}.yml"
+        - "{{ ansible_distribution }}_{{ ansible_distribution_version }}.yml"
```
Should this also switch? It would require changing `Ubuntu_16.04.yml` as well.
```suggestion
        - "{{ ansible_distribution }}_{{ ansible_distribution_release }}.yml"
```

jsf9k replied:
Thanks!  I noticed that this morning too.

mcdonnnj reviewed (approved):
Makes sense. Please see my one comment about a change you made in https://github.com/cisagov/ansible-role-xfce/pull/10.

jsf9k committed:
Prefer ansible_distribution_release to ansible_distribution_version

Some distributions (like Debian) can have multiple version numbers that point to the same release.

Co-authored-by: Nick M. <50747025+mcdonnnj@users.noreply.github.com>
